specs/agents/tracing-spans-drop-fast-exit.md

```diff
@@ -0,0 +1,81 @@
+[Agent spec home](README.md) > [Handling huge traces](tracing-spans-handling-huge-traces.md) > [Dropping fast exit spans](tracing-spans-drop-fast-exit.md)
+
+# Dropping fast exit spans
+
+If an exit span was really fast, chances are that it's not relevant for analyzing latency issues.
+Therefore, agents SHOULD implement the strategy laid out in this section to let users choose the level of detail/cost tradeoff that makes sense for them.
+If an agent implements this strategy, it MUST also implement [Collecting statistics about dropped spans](tracing-spans-dropped-stats.md).
+
+## `exit_span_min_duration` configuration
+
+Sets the minimum duration of exit spans.
+Exit spans that execute faster than this threshold are attempted to be discarded.
+
+The attempt fails if they lead up to a span that can't be discarded.
```

Comment:
Can you clarify what "they lead up to a span that can't be discarded" means here? Is it if a child span cannot be discarded?

Reply:
```suggestion
In some cases exit spans cannot be discarded.
```